src/modules/quizzes.js

```diff
@@ -164,11 +170,12 @@ class Quizzes {
    *
    * @function getQuizResults
    * @description Retrieve quiz recommendation and filter expression from Constructor.io API
-   * @param {string} id - The identifier of the quiz
+   * @param {string} quizId - The identifier of the quiz
    * @param {string} [parameters] - Additional parameters to refine result set
```

Comment:
Since a child of `parameters` is required can we also remove [] around parameters here?

Reply:
Agreed, let's update this